### PR TITLE
Peer Pinging Tornado Fixes

### LINKED ISSUE #1233: Endless page refresh loop after clicking a store.  Mac 10.10.3, safari and chrome.
After clicking on peer, the page goes into an auto-refresh mode where it's almost as if the refresh button is being clicked a couple times per second.

I get this http://pastebin.com/2KhqaC7M over and over again in my browser console while it's auto refreshing.

This is with a clean install of openBazaar.  Deleting the database and running configure again didn't help.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/11732788-endless-page-refresh-loop-after-clicking-a-store-mac-10-10-3-safari-and-chrome?utm_campaign=plugin&utm_content=tracker%2F1516753&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F1516753&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by hoffmabc**
Also Fixes #1233

### CODE REVIEW

**Comment by hoffmabc** [comment]
@gubatron this fixes the constant that you added I believe because it is specified in milliseconds and not seconds so it was not working properly.

**Comment by gubatron** [comment]
Will review tonight

File: node/connection.py [hunk 1]
```diff
@@ -130,8 +130,7 @@ def pinger():
 
                     # yappi.get_thread_stats().print_all()
 
-        self.ping_task = ioloop.PeriodicCallback(pinger, PEERCONNECTION_PING_TASK_INTERVAL_IN_MS, io_loop=self.loop)
-        self.ping_task.start()
+        self.loop.call_later(PEERCONNECTION_PING_TASK_INTERVAL_IN_SECONDS, pinger)
```
Comment (gubatron) [inline node/connection.py]: cool. Better like this keeping everything in seconds.

Wonder why ioloop.PeriodicCallback() takes ms, and this one takes seconds... API inconsistencies suck. If it were up to me I'd have everything in MS for more control.

File: node/constants.py [hunk 2]
```diff
@@ -69,7 +69,7 @@
 PEERCONNECTION_NO_RESPONSE_DELAY_IN_SECONDS = 10
 PEERCONNECTION_SENDING_OUT_DELAY_IN_SECONDS = 5
 PEERCONNECTION_PINGER_TIMEOUT_IN_SECONDS = 30
-PEERCONNECTION_PING_TASK_INTERVAL_IN_MS = 5000
+PEERCONNECTION_PING_TASK_INTERVAL_IN_SECONDS = 5
```
Comment (gubatron) [inline node/constants.py]: :+1:

File: node/ws.py [hunk 3]
```diff
@@ -1374,7 +1374,7 @@ def get_peers(self):
                 peer.reachable = True
                 reachable_count += 1
 
-            if hasattr(peer, 'hostname') and peer.guid:
+            if hasattr(peer, 'hostname') and peer.guid[:4] != 'seed':
```
Comment (gubatron) [inline node/ws.py]: all seed servers have `seed` as their prefix on the guid? interesting.

Comment (gubatron) [inline node/ws.py]: in the future such distinctions should be made as a single flag, on a field where we can put a bunch of flags.

Comment (Renelvon) [inline node/ws.py]: Indeed. We will, soon. :)

Comment (hoffmabc) [inline node/ws.py]: Yeah this is absolutely a hack and hopefully @Renelvon has something up his sleeve on this one.

Comment (gubatron) [inline node/ws.py]: :+1:

**Pull request merged**